AWS/legos/aws_list_expiring_access_keys/aws_list_expiring_access_keys.py

```diff
@@ -17,9 +19,14 @@ class InputSchema(BaseModel):
 def aws_list_expiring_access_keys_printer(output):
     if output is None:
         return
-    pprint.pprint(output)
+    if isinstance(output, CheckOutput):
+        pprint.pprint(output.status)
```

Comment:
We should print the json.dumps of the object, so that backend code can just do unmarshal and get the fields

Reply:
This will be handled by @amit-chandak-unskript